internal/server/ui/index.html

```diff
@@ -1,62 +1,84 @@
-<html>
+<!DOCTYPE html>
+<html lang="en">
 
 <head>
   <title>DDNS Updater</title>
   <link rel="icon" href="/favicon.ico" type="image/x-icon">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
   <style>
     table {
-      font-family: arial, sans-serif;
-      font-size: 14px;
-      font-size: 1vw;
       border-collapse: collapse;
+      background: white;
+      border-radius: 6px;
+      overflow: hidden;
+      max-width: 1200px;
       width: 100%;
+      margin: 0 auto;
+      position: center;
     }
-
-    td,
-    th {
-      border: 2px solid #9a9fa1;
-      text-align: center;
-      padding: 1%;
-      max-width: 35%;
-      transition: all 0.7s;
+    table * {
+      position: relative;
     }
-
-    th {
-      background-color: #d8daf7;
+    table td, table th {
+      padding-left: 8px;
+    }
+    table thead tr {
+      height: 60px;
+      background: #FFED86;
+      font-size: 16px;
+    }
+    table tbody tr {
+      height: 48px;
+      border-bottom: 1px solid #E3F1D5;
+    }
+    table tbody tr:last-child {
+      border: 0;
+    }
+    table td, table th {
+      text-align: center;
     }
 
-    tr:nth-child(odd) {
-      background-color: #e6f7ea;
+    .footer {
+      position: fixed;
+      bottom: 0;
+      text-align: center;
+      width: 100%;
+      height: 65px;
     }
 
-    tr:nth-child(even) {
-      background-color: #f3ebe3;
+    a, b {
+      color: darkcyan;
+      text-decoration: none;
     }
 
-    tr {
-      transition: all 0.7s;
+    a.footer {
+      padding-top: 5px;
     }
 
-    tr:hover {
-      background: #c1e2f0;
+    body {
+      background: #2b2b2b;
+      font: 400 14px "Monospaced", "sans-serif";
+      padding: 20px;
     }
 
-    a {
-      text-decoration: none;
+    blockquote {
+      color: white;
+      text-align: center;
     }
   </style>
 </head>
 
 <body>
+
   <table>
     <tr>
       <th>Domain</th>
       <th>Host</th>
       <th>Provider</th>
-      <th>IP version</th>
-      <th>Update status</th>
-      <th>Set IP</th>
-      <th>Previous IPs (reverse chronological order)</th>
+      <th>IP Version</th>
+      <th>Update Status</th>
+      <th>Current IP</th>
+      <th>Previous IPs</th>

```

Comment:
Can we keep the `(reverse chronological order)` somehow? :thinking:

Reply:
Yeah we can, it just looked cluttered and seems obvious in my opinion

Reply:
Maybe we could make it show on hover, that would probably be the best of both worlds